src/main/java/io/temporal/samples/helloworld/HelloWorldStarter.java

```diff
@@ -0,0 +1,31 @@
+package io.temporal.samples.helloworld;
+// @@@SNIPSTART java-hello-world-sample-workflow-starter
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class HelloWorldStarter {
+
+  public static void main(String args[]) throws Exception {
+
+    // gRPC stubs wrapper that talks to the local docker instance of temporal service.
+    WorkflowServiceStubs service = WorkflowServiceStubs.newInstance();
+    // client that can be used to start and signal workflows
+    WorkflowClient client = WorkflowClient.newInstance(service);
+    // Define the name of the Task Queue that the Workflow and Activity Tasks will be sent to
+    final String TASK_QUEUE = "java-hello-world-task-queue";
+    // Start a workflow execution.
+    // Uses task queue from the GreetingWorkflow @WorkflowMethod annotation.
+    HelloWorldWorkflow workflow =
+        client.newWorkflowStub(
+            HelloWorldWorkflow.class,
+            WorkflowOptions.newBuilder().setTaskQueue(TASK_QUEUE).build());
+    // Use the Workflow method to get the
```

Comment:
Change to:

// Starts workflow execution and waits for its completion. If this process dies the workflow still continues execution assuming that the worker process is up and running.